plutus-tx-plugin/test/Plugin/Typeclasses/fmapDefaultTest.plc.golden

```diff
@@ -20,6 +20,23 @@
       )
       addInteger
     )
+    (termbind
+      (strict)
+      (vardecl
+        bad_name
```

Comment:
This `bad_name` is `+`. In the GHC 9.2.4 core, there is an additional let-binding (`v_B1`) that wraps `+`, and I think this is where it comes from.

GHC 8.10.7 Core:
```
src<test/Plugin/Typeclasses/Spec.hs:118:50-98>	
fmapDefault	
  @ []	
  @ Integer	
  @ Integer	
  $fTraversable[]	
  (let {	
     ds_dnn6 [Occ=OnceL1] :: Integer	
     [LclId,	
      Unf=Unf{Src=<vanilla>, TopLvl=False, Value=True, ConLike=True,	
              WorkFree=True, Expandable=True, Guidance=IF_ARGS [] 100 0}]	
     ds_dnn6 = src<test/Plugin/Typeclasses/Spec.hs:118:70> 1 } in	
   \ (ds_dnn5 [Occ=Once1] :: Integer) ->	
     src<test/Plugin/Typeclasses/Spec.hs:118:65-71>	
     + @ Integer $fAdditiveSemigroupInteger ds_dnn5 ds_dnn6)	
  (src<test/Plugin/Typeclasses/Spec.hs:118:73-97>	
   build	
     @ Integer	
     (\ (@ a)	
        (c_dnn8 [OS=OneShot] :: Integer -> a -> a)	
        (n_dnn9 [Occ=Once1, OS=OneShot] :: a) ->	
        c_dnn8	
          (src<test/Plugin/Typeclasses/Spec.hs:118:74-87> 1)	
          (c_dnn8	
             (src<test/Plugin/Typeclasses/Spec.hs:118:90> 2)	
             (c_dnn8	
                (src<test/Plugin/Typeclasses/Spec.hs:118:93> 3)	
                (c_dnn8 (src<test/Plugin/Typeclasses/Spec.hs:118:96> 4) n_dnn9)))))
```

GHC 9.2.4 Core:

```
src<test/Plugin/Typeclasses/Spec.hs:118:50-98>
fmapDefault
  @[]
  @Integer
  @Integer
  $fTraversable[]
  (let {
     v_B1 [Occ=OnceL1!] :: Integer -> Integer -> Integer
     [LclId,
      Unf=Unf{Src=<vanilla>, TopLvl=False, Value=False, ConLike=False,
              WorkFree=False, Expandable=True, Guidance=IF_ARGS [] 20 0}]
     v_B1
       = src<test/Plugin/Typeclasses/Spec.hs:118:66-68>
         + @Integer $fAdditiveSemigroupInteger } in
   let {
     v_B3 [Occ=OnceL1] :: Integer
     [LclId,
      Unf=Unf{Src=<vanilla>, TopLvl=False, Value=True, ConLike=True,
              WorkFree=True, Expandable=True, Guidance=IF_ARGS [] 100 0}]
     v_B3 = src<test/Plugin/Typeclasses/Spec.hs:118:70> 1 } in
   \ (v_B2 [Occ=Once1] :: Integer) ->
     src<test/Plugin/Typeclasses/Spec.hs:118:65-71> v_B1 v_B2 v_B3)
  (src<test/Plugin/Typeclasses/Spec.hs:118:73-97>
   build
     @Integer
     (\ (@a)
        (c_dkgj [OS=OneShot] :: Integer -> a -> a)
        (n_dkgk [Occ=Once1, OS=OneShot] :: a) ->
        c_dkgj
          (src<test/Plugin/Typeclasses/Spec.hs:118:74-87> 1)
          (c_dkgj
             (src<test/Plugin/Typeclasses/Spec.hs:118:90> 2)
             (c_dkgj
                (src<test/Plugin/Typeclasses/Spec.hs:118:93> 3)
                (c_dkgj (src<test/Plugin/Typeclasses/Spec.hs:118:96> 4) n_dkgk)))))
```